client/blocks/reader-subscription-list-item/email-settings.jsx

```diff
@@ -106,5 +151,29 @@ class ReaderEmailSubscriptionSettingsPopout extends Component {
 	}
 }
 
-export default localize( ReaderEmailSubscriptionSettingsPopout );
+const mapStateToProps = ( state, ownProps ) => {
+	const follow = find( getReaderFollows( state ), { blog_ID: ownProps.siteId } );
+
+	const currentDeliveryMethods = get( follow, [ 'delivery_methods', 'email' ], {} );
```

Comment:
yeah, it was the defaulting that i liked about it. get + shallow destructuring ftw